examples/react/package.json

```diff
@@ -11,6 +11,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "loki",
+    "pretest-ci": "npm run update-loki",
```

Comment:
What is this command about? I can't run it here

Reply:
It's loki repo specific, to make sure the tests run the loki code from that specific branch/commit